index.html

```diff
@@ -231,6 +231,27 @@ <h3 class="best" id="skill">
         <section class="contact">
             <p class="paragraph1">I'm always interested in hearing about new projects, so if you'd like to chat please get
                 in touch.</p> 
-    </div>
+        <form action="https://formspree.io/f/mgeqroja" method="POST" class="form-container">
+            <label for="name">
+                <input type="text" name="name" placeholder="Full name" maxlength="30" required>
+            </label>
+            <label for="email">
+                <input type="email" name="email" placeholder="Email address"  required>
+            </label>
+            <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+        <button class="btn">Get in Touch</button>
+        </form>
+    </section>
+    <footer>
+        <ul class="social-container">
+            <li class="social"><a href="#"><img src="./images/git.svg"></img></a></li>
+            <li class="social"><a href="#"><img src="./images/linkedin.svg" class="linkedin"></img></a></li>
+            <li class="social"><a href="#"><img src="./images/v.svg"></img></a></li>
+            <li class="social"><a href="#"><img src="./images/twitter.svg"></img></a></li>
+            <li class="social"><a href="#"><img src="./images/M.svg"></img></a></li>
```

Comment:
Well done in adding the images. I would suggest that you could add an alt tag to it. This is so that when your images does not load it displays an alt message. Kindly add them in.